src/plasmapy/utils/decorators/checks.py

```diff
@@ -1174,9 +1171,6 @@ def foo(arg1):
         )
         def foo(arg1):
             return arg1
-
-    .. _astropy equivalencies:
```

Comment:
Docs already referenced this link in a number of places, so I changed it to the global reference.